clang/test/SemaHLSL/BuiltIns/ddx-coarse-errors.hlsl

```diff
@@ -0,0 +1,21 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -verify
```

Comment:
nit: those are sema tests so should be the same but doesn't cost to also add a SPIR-V run line (same for ddy)